### PR TITLE
Small mobile style cleanups

### PR DESCRIPTION
**Pull request by sambrannon**
This PR adds Styled Components' `createGlobalStyle` to add some basic global styles to the app...

1. `* { box-sizing: border-box; }` resets the calculations for elements in the app, making it much easier to properly size things. [More info](https://css-tricks.com/box-sizing/#present-day-box-sizing)
2. Margin and padding resets on top-level elements. Just to get rid of some default spacing.
3. Removed the 600px min width (since that is what was causing the overflow on smaller screens. Replaced it with a media query to make sure those columns wrap once you get smaller than 768px.

### CODE REVIEW

**Comment by brannonjames** [comment]
Looks good - I'll look into the failing Firebase Auth in the Action

**Review by brannonjames — APPROVED** [review]

**Comment by brannonjames** [comment]
*Finally* figured this on out - this looks to be the issue

https://stackoverflow.com/questions/58737785/github-actions-empty-env-secrets

TLDR: You can't access the repo secrets when opening a PR from a fork (unless you have write access)
